src/gallery-pane.js

```diff
@@ -492,21 +539,27 @@
 
                 var self = this;
 
-                return '<div class="' + data.elementClassName + '">' +
+                return '<div class="' + ((data.shouldShowSidebarAd()) ? ('hasSidebarBanner ' + data.elementClassName) : data.elementClassName) + '">' +
```

Comment:
@amikolic 
rewrite this to more readable format such as this:
"' + (data.shouldShowSidebarAd() ? 'hasSidebarBanner ' : '') + data.elementClassName + '"